index.js

```diff
@@ -44,4 +39,45 @@ const nextOnNetlify = (options = {}) => {
   trackNextOnNetlifyFiles();
 };
 
+const watch = (functionsPath, publishPath) => {
+  logTitle(`👀 Watching source code for changes`);
+
+  const watcher = chokidar.watch(SRC_FILES);
+  const runBuild = debounceFn(
+    () => {
+      try {
+        execa.sync("next", ["build"], { stdio: "inherit" });
```

Comment:
Is there a better way to do this?
I tried setting `watch` on the webpack config per https://github.com/vercel/next.js/issues/7528#issuecomment-502141153 but that didn't work

Reply:
:/ yeah i see what youre sayin, i think this is the best we can do